File: guessing.pl
#!/bin/perl -w

use warnings;

$min = 100;
$max = 200;
$guess = -1;
$random = int(rand($max - $min)) + $min;
$count = 0;
$allowedGuesses = 7;

print "Guess a number between 100 and 200.  You are allowed 7 guesses.\n";

while ($guess != $random && $count < $allowedGuesses) {

	$count++;
	print "Enter guess $count: ";
	$guess = <STDIN>;

	if ($guess > $random) {
		print "high guess\n";
	}
	elsif ($guess < $random) {
		print "low guess\n";
	}
}

if ($guess == $random) {
	print "Correct!  That took $count guesses.\n";
} else {
	print "Sorry!  You are only allowed 7 guesses.\n";
}

print "Enter your name: ";
my $name = <STDIN>;

open(my $fh, '>>', 'highscores.txt');
print %fh "$name - $count guesses";

File: guessing.py
#Guessing Game

import random

randomNumber = random.randint(-100,100)
guess = -1
count = 0
allowedGuesses = 7

print ("Guess a number between 100 and 200.  You are allowed 7 guesses.")

while guess != randomNumber and count < allowedGuesses:

    count = count + 1
    countString = str(count)
    print ("Enter guess " + countString + ": ")
    guess = input()
    guess = int(guess)

    if guess > randomNumber:
        print ("high guess")
    if guess < randomNumber:
        print ("low guess")
    
if guess == randomNumber:
    print ("Correct!  That took " + countString + " guesses.")
else:
    print ("Sorry!  You are only allowed 7 guesses.")

print("Enter your name: ")
name = input()
name = str(name)

f = open("gameReport.txt", "w")
f.write(name + " - " + countString + " guesses\n")
f.close()





    
